tests/unit/test_simulator.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from model import storage
+from model.simulator import METHODS, simulate
+
+
+def test_tmy_result():
+    tmy_objectives = ['bigg:akg']
+    to_return = ['fluxes', 'tmy', 'model', 'growth-rate', 'removed-reactions']
+    result = simulate(storage.get('iJO1366').model, 'fba', None, None, tmy_objectives, to_return)
+    assert set(result.keys()) == set(to_return)
```

Comment:
When building a list or set of the keys of a dictionary there's no need to use the `.keys()` method since the default iterator over a dictionary is over the keys.